sw/device/silicon_creator/manuf/base/tpm_personalize_ext.c

```diff
@@ -135,7 +135,8 @@ static status_t personalize_gen_tpm_certificates(
 
 status_t personalize_extension_pre_cert_endorse(
     ujson_t *uj, manuf_certgen_inputs_t *certgen_inputs,
-    manuf_certs_t *tbs_certs, cert_flash_info_layout_t *cert_flash_layout) {
+    manuf_certs_t *tbs_certs, cert_flash_info_layout_t *cert_flash_layout,
+    dif_flash_ctrl_state_t *flash_ctrl_handle) {
```

Comment:
You will need to `#include "sw/device/lib/dif/dif_flash_ctrl.h"` at the top for this to compile. Same for the other extension files here.